packages/aws-lambda/test/integration_test/test_definition.js

```diff
@@ -1039,7 +1067,7 @@ function registerTests(handlerDefinitionPath) {
       trigger: 'api-gateway-proxy',
       instanaEndpointUrl: backendBaseUrl,
       instanaAgentKey,
-      serverTiming: 'string'
+      serverTiming: true
```

Comment:
FYI: Removed the array case, because we already test adding `response.multiValueHeaders`. No need to test that twice.